include/net/ethernet.h

```diff
@@ -673,6 +673,26 @@ int net_eth_promisc_mode(struct net_if *iface, bool enable);
  */
 struct device *net_eth_get_ptp_clock(struct net_if *iface);
 
+/**
+ * @brief Return PTP clock that is tied to this ethernet network interface
+ * index.
+ *
+ * @param index Network interface index
```

Comment:
Please describe numeric base of the index (we standardized on POSIX base of 1).

Reply:
We are referring to network index in lot of APIs. The range of the values can be added to general documentation, not to individual API documentation.

Reply:
I'd personally suggest to repeat it everywhere :-I. But first of all, it's a ping to you to verify if you consistently use 1-based index.